test/e2e/performanceprofile/functests/utils/cgroup/config/config.go

```diff
@@ -0,0 +1,20 @@
+package config
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const CgroupMountPoint = "/sys/fs/cgroup"
+
+type Config struct {
+	CPUSet string
+	Quota  string
+	Period string
+}
```

Comment:
Can we define structure in a more hierarchial fashion i.e: Controllers->Controller-interfacefiles

```
type CgroupController struct {
          //suported cgroup Controllers
           cpuset
           cpu
           // any future cgroup controllers 
   }
```
   Then created methods that return interface files of that particular controller ?